### DISCUSSION TITLE
Ubuntu with chia 1.3.5 stuck after 30 hours running

### DISCUSSION THREAD

**Discussion by Smoko-Time** [Chia-Network/chia-blockchain#11744] [Support]
After the upgrade to 1.3.5, a have some issues with chia farming and Ubuntu 22.04.

I don't make the update from 1.3.4 to 1.3.5. I format the Ubuntu and make a new installation of Chia. New Synchronization. After 3 days of download, all files and I started farming... I'm happy because all things are working...
But, between 30 and 40 hours after restarting the Ubuntu the Chia close the software and the computer is slower.

I looked at the memory usage and I think is too high. There are 2 photos, 1 when I restart the Ubuntu and Chia are running great only 4Gb memory usage, and the 2 when the application was closed, and the memory usage is more than 13GB.
Photo 1
![1](https://user-images.githubusercontent.com/106654210/171577692-9975ed2a-0380-466a-9f3f-f360884efa9d.png)

Photo 2
![2](https://user-images.githubusercontent.com/106654210/171577772-48bcefe6-dd75-4973-b6d7-2b3b39365616.png)

Photo 3 are with 24hs running.
![3](https://user-images.githubusercontent.com/106654210/171577803-89658b0f-899a-4838-96e8-891a4b5a1019.png)

This is a problem in Chia or Ubuntu?
I have checked the memory card and inverter it, but not changed.
It starts only after the update....

**Comment by loppefaaret** [comment]
Im not familiar with the ubuntu system overview that you are showing in your pictures, but it is normal for an OS to keep the memory close to filled up, rather than discarding unused memory. in case it needs the same data again at some point, it already have it in memory, rather than needing to recreate it... should the OS need the memory that is unused, but allocated, it will release it, and fill in the new data into memory...

**Reply by Smoko-Time** [reply]
Suppose it is not a problem in memory usage. Is a problem with ubuntu installation?
Because when the system is stuck. The Ubuntu requested the password login. But the machine was not restarted, and after making the login the Ubuntu is soo slow and I need to restart it back to work normally.

But What are my options? Format and install Chia and Ubuntu at zero again? Use the old version of Ubuntu?
